biofefi/feature_importance/call_methods.py

```diff
@@ -216,7 +216,7 @@ def save_target_clusters_plots(
     # Plot boxplot of the target (continuous values) and target clusters (categories) using seaborn
     plt.style.use(plot_opt.plot_colour_scheme)
     fig, ax = plt.subplots(layout="constrained")
-    sns.boxplot(data=df_cluster, x="cluster", y="target", ax=ax)
+    sns.boxplot(data=df_cluster, x="cluster", y="target", hue="cluster", ax=ax)
```

Comment:
May I know why `hue="cluster" `is used here? Since the x-axis already categorises by cluster, it might not add new information and could be redundant if the same data is being overridden. 

If the goal is to colour boxes differently while still displaying categories on the x-axis, we can also use a `palette` parameter to achieve this. 

For eg: 
`sns.boxplot(data=df_cluster, x="cluster", y="target", ax=ax, palette="bright")`

- If` hue="cluster"`  is retained for specific visual goals such as grouping within clusters, then this approach works well and feel free to keep it as it is.